### PR TITLE
Update 21d2b384f4fb5317192586effb3269060e8164b13ce89888070cac974a4541…

### PR DESCRIPTION
**Pull request by Liberty-Chris**
…4c4f5553.json

Update Jealous Decimals

# Pull Request Template

## Description

Please include a short summary of the changes in this PR.

## Type of change

- [ ] Metadata related change
- [x] Other

## Checklist:

- [ ] For metadata related changes, this PR code passes the GitHub Actions metadata validation

## Metadata PRs

Please note it may take up to 4 hours for merged changes to take effect on the metadata server.

### CODE REVIEW

**Comment by Liberty-Chris** [comment]
This is to update JEALOUS decimals from 0 to 6.

**Comment by Godspeed-exe** [comment]
**Automated feedback**<br /><br />**Value:** <br />I am the Joker to GUMMY & BAKED's Batman & Robin... Yeah I'm a Jealous Meme coin, but I have reason to be... let's get a real Gummy for the ADA community and anyone else who wants to stick it to the man Twitter/X - https://x.com/TheJealousGummy TG - https://t.me/+j54plk-e5R1lYmJk website - https://jealousgummy.online/cryptomemecoin<br /><br />**Attribute:** <br />description<br /><br />**Analysis:** <br />This value makes references to popular characters like the Joker, Batman, and Robin, which are copyrighted properties owned by DC Comics. Additionally, it references a 'Gummy' meme coin tied to ADA (Cardano's cryptocurrency) which could potentially infringe on established trademarks and brands within the crypto ecosystem.<br /><br />

**Comment by Godspeed-exe** [comment]
hello @Liberty-Chris 

Don't mind that automatic feedback. The PR does seem to fail the automated metadata verification because the sequenceNumber isn't incremented. When submitting updates, this needs to be incremented for the attributes which are changing. 

Does that make sense?

**Comment by Liberty-Chris** [comment]
@Godspeed-exe I'm looking into this now. I apologize for the delay. I've been sick for a bit.

File: mappings/21d2b384f4fb5317192586effb3269060e8164b13ce89888070cac974a45414c4f5553.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     },
     "decimals": {
         "sequenceNumber": 0,
-        "value": 0,
```
Comment (Liberty-Chris) [inline mappings/21d2b384f4fb5317192586effb3269060e8164b13ce89888070cac974a45414c4f5553.json:35]: 1

**Comment by Liberty-Chris** [comment]
@Godspeed-exe I submitted what I think is correct. If something is still incorrect, please give me some specifics. I'm trying to assist a DDz client with this update and I am sick and fog brained. My apologies again and I appreciate you.

**Comment by Godspeed-exe** [comment]
hey @Liberty-Chris 

The Validate-Metadata job is still failing with below error message

` [Main.log#290] Failed to validate wallet metadata entry, error was: 'attestation verification failed for: decimals'.`

If you change values, you have to resign them with the policy.skey . does that make sense? hope you feel better man!

**Comment by Liberty-Chris** [comment]
What do I need to change for this update to succeed?

**Comment by Godspeed-exe** [comment]
Please have a look at [Updating an entry](https://github.com/cardano-foundation/cardano-token-registry/wiki/How-do-I-update-my-entry-in-the-registry%3F) on the Wiki page. Specifically Step5 which talks about the signing process.

So I would:
- close this PR
- go through the signing process
- commit to your fork
- do a new PR

Or you can also do a new commit to this PR, depends on how familiar you are with Github.

**Comment by Godspeed-exe** [comment]
The automatic process checks if the values are done by the owner, because Github is open to everyone, everyone could try to do an update for a token. But this check verifies that it's the actual owner (who holds the keys to the policy) that's doing the update.

**Comment by Godspeed-exe** [comment]
Automatically closed because it's been more than 5 days since the last comment

**Pull request closed without merging**